solution_1.rb

```diff
@@ -0,0 +1,91 @@
+class Tutorial
+
+  def solution(x)
```

Comment:
Optional: maybe add the question name (eg. `diverse_word_solution`) instead of this. Or you can use 1 class per question (eg. `class DiverseWord` and `def solution`). It's easier to understand.